package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "02-tools_and_context",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) 02: Tools and Context ======",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
should be running mocha with this script. Also need to add lint script should be "eslint ."